core/app/models/mno_enterprise/user.rb

```diff
@@ -1,59 +1,43 @@
-# == Schema Information
-#
-# Endpoint:
-#   - /v1/users
-#   - /v1/organizations/:organization_id/users
-#
-#  id                             :string          e.g.: 1
-#  uid                            :string          e.g.: usr-k3j23npo
-#  email                          :string(255)     default(""), not null
-#  authenticatable_salt           :string(255)     used for session authentication
-#  encrypted_password             :string(255)     default(""), not null
-#  reset_password_token           :string(255)
-#  reset_password_sent_at         :datetime
-#  remember_created_at            :datetime
-#  sign_in_count                  :integer         default(0)
-#  current_sign_in_at             :datetime
-#  last_sign_in_at                :datetime
-#  current_sign_in_ip             :string(255)
-#  last_sign_in_ip                :string(255)
-#  confirmation_token             :string(255)
-#  confirmed_at                   :datetime
-#  confirmation_sent_at           :datetime
-#  unconfirmed_email              :string(255)
-#  failed_attempts                :integer         default(0)
-#  unlock_token                   :string(255)
-#  locked_at                      :datetime
-#  created_at                     :datetime        not null
-#  updated_at                     :datetime        not null
-#  name                           :string(255)
-#  surname                        :string(255)
-#  company                        :string(255)
-#  phone                          :string(255)
-#  phone_country_code             :string(255)
-#  geo_country_code               :string(255)
-#  geo_state_code                 :string(255)
-#  geo_city                       :string(255)
-#  website                        :string(255)
-#  api_key                        :string(255)
-#  api_secret                     :string(255)
-#
-
 module MnoEnterprise
   class User < BaseResource
-    include MnoEnterprise::Concerns::Models::IntercomUser if MnoEnterprise.intercom_enabled?
+    include MnoEnterprise::Concerns::Models::IntercomUser
```

Comment:
It could stay, but I had a lot of trouble making specs works because of it (don't really remember why)
And I thought that could be simplified. That is indeed debatable.